notebooks/week6/01.send_request_to_endpoint.py

```
@@ -0,0 +1,135 @@
+                        .toPandas()
+
+test_set = spark.table(f"{catalog_name}.{schema_name}.test_set_nico") \
+                        .withColumn(config.id_col, col(config.id_col).cast("string")) \
+                        .toPandas()
+
+
+# COMMAND ----------
+
+try:
+    # notebook way
+    token = dbutils.notebook.entry_point.getDbutils().notebook().getContext().apiToken().get()
```

Ruff (0.7.0)

Undefined name `dbutils`

(F821)